src/ducks/connections/index.js

```diff
@@ -240,6 +244,13 @@ const triggersReducer = (state = {}, action) => {
             }
           }, state)
         : state
+    case RECEIVE_DELETED_DOCUMENT:
+      if (
+        action.updateCollections &&
```

Comment:
`action.updateCollections` is not designed to be used like this. Instead I suggest to do:
```
const { data } = action.response
const { _id, _type } = data[0]
if(data._type === TRIGGERS_DOCTYPE) {
  return omit(state, data._id)
}
```